features/support/fs.rs

```diff
@@ -97,6 +101,10 @@ pub fn create_project() -> io::Result<Project> {
 }
 
 fn create_cargo_toml(dir: TempDir) -> io::Result<TempDir> {
+  fn double_backslash<T: Display>(path: T) -> String {
```

Comment:
This is a pretty weird method:

- Is there a reason its defined within create_cargo_toml? I'm not entirely opposed to the idea but I'm not sure what the current suggested best practice is regarding inner methods.

- I'm curious why this method was defined the way it was, taking a T that satisfies Display rather than just whatever the type at line 123 is. String? str&? Not really a big deal

No big deal on either account.